website/docs/r/netlify_build_hook.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: "netlify"
+page_title: "Netlify: netlify_build_hook"
+sidebar_current: "docs-netlify-resource-build-hook"
+description: |-
+  Provides an build hook resource.
+---
+
+# netlify_build_hook
+
+Manages build hooks, also known as [incoming webhooks](https://www.netlify.com/docs/webhooks/#outgoing-webhooks). These can, at the time of writing, only be used to trigger new builds of the site. To create one, provide your site id along with the name of the hook, and the branch to be built when the hook is triggered.
```

Comment:
I believe there is an 80 or so character limit for markdown files so we should make this one line multiple lines